misc/cargo-vet/imports.lock

```diff
@@ -22,6 +22,13 @@ user-id = 160111
 user-login = "aws-sdk-rust-ci"
 user-name = "AWS SDK Rust Bot"
 
+[[publisher.aws-config]]
+version = "1.2.1"
+when = "2024-04-22"
+user-id = 160111
+user-login = "aws-sdk-rust-ci"
+user-name = "AWS SDK Rust Bot"
+
```

Comment:
We should remove the old versions from this file.

Reply:
I assumed `cargo vet check` would do that automatically but apparently not!